source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+- Users enter URL to your hosted Mattermost Server.
+- Extend your organization’s security best practices/requirements via your EMM provider.
+
+**Advantages:** Easy, mobile deployment can be done by each individual user, app security and configuration can be maintained via your EMM provider 
+**Disadvantages:** No app customization
+
+***********************************************
+Distributing Custom Versions of the Mobile Apps
+***********************************************
+
+**Desired Outcomes:**
+
+- Maintain full control over the distribution of applications
+- Change the look, feel, and capabilities of the Mattermost mobile application.
+- Limit message traffic from passing through the HPNS
+
+This model is more difficult, and is recommended for organizations that can't (or don't desire) to use the HPNS. Often security and access requirements determine this, not the size of your organization.
+
+Both Google and Apple require signatures of the application and push proxy to match. This means if you build the applications, you must host your own instance of MPNS.
+
+Building the apps can be an involved process. This requires that you have the skillset to maintain and deploy packaged apps. Part of this process includes, but is not limited to :
+
+- Obtaining/providing certificates for your custom application
+- Signing your custom applications
+- Distributing your applications via public or private app stores
+
+Your custom built apps are a fork of the Mattermost project. This means you will be responsible for adding in any feature or security updates.  
+
+To understand what’s involved, have your development team `read through our documentation <https://developers.mattermost.com/contribute/mobile/build-your-own/>`_.
+
+    **Note:** As of Mattermost 5.18, E20 customers can limit data sent to the HPNS. With this option a message containing only an ID. Once the mobile client receives this ID, the message contents are loaded from the server. Thus, the message contents are never transmitted through APNS/FCM.
+
+----
+
+#####################################################
+Mobile Deployment via Public App Stores (Recommended)
+#####################################################
+
+To proceed you must have a Mattermost Server installed and accessible. This is also true for  using the hosted version of the Mattermost push proxy (HPNS).
+
+*********************************
+Accessing the Mobile Applications
+*********************************
+
+The Mattermost mobile application is available for both Android and iOS devices. At this point, it’s as simple as having your users download the application and `point to your Mattermost Server URL <https://docs.mattermost.com/help/getting-started/signing-in.html#ios-setup>`_.
+
+    **Note:** The Mattermost Mobile apps are signed and have certificates associated with Mattermost and the public app stores. This means they will not work if you are privately hosting the Mattermost Push Proxy Service.
```

Comment:
public apps stores mentioned again....

Reply:
The beatings will continue... until morale improves.

Reply:
resolved in meeting